libs/designer-ui/src/lib/settings/settingsection/settingTokenField.tsx

```diff
@@ -376,6 +377,18 @@ export const TokenField = ({
         />
       );
 
+    case constants.PARAMETER.EDITOR.MIXEDINPUTEDITOR: {
+      return (
+        <MixedInputEditor
+          supportedTypes={editorOptions?.supportedTypes}
+          useStaticInputs={editorOptions?.useStaticInputs}
+          initialValue={value}
+          isRequestApiConnectionTrigger={editorOptions?.isRequestApiConnectionTrigger}
+          onChange={onValueChange || (() => {})}
```

Comment:
The more modern appropriate operator in Javascript for what you're trying to do here is now `??` instead of `||`

Additionally, what you could do in your component instead of short circuiting here to a default, is something like

```typescript
onChange?.({ value });
```

That's how you can safely call `onChange`, even if it's `null` or `undefined`

Reply:
Ah, thanks for the suggestion. Updated to use ?? and added the `onChange?.()` on the component